lib/console.js

```diff
@@ -35,46 +35,38 @@ function Console(stdout, stderr) {
   }
 }
 
-Console.prototype.log = function() {
+Console.prototype.log = function () {
   this._stdout.write(util.format.apply(null, arguments) + '\n');
 };
 
-
 Console.prototype.info = Console.prototype.log;
 
-
-Console.prototype.warn = function() {
+Console.prototype.warn = function () {
   this._stderr.write(util.format.apply(null, arguments) + '\n');
 };
 
-
 Console.prototype.error = Console.prototype.warn;
 
-
-Console.prototype.dir = function(object, options) {
+Console.prototype.dir = function (object, options) {
   this._stdout.write(util.inspect(object, util._extend({
-    customInspect: false
-  }, options)) + '\n');
+      customInspect: false
+    }, options)) + '\n');
 };
 
-
-Console.prototype.time = function(label) {
+Console.prototype.time = function (label) {
   this._times.set(label, process.hrtime());
 };
 
-
-Console.prototype.timeEnd = function(label) {
-  var time = this._times.get(label);
-  if (!time) {
```

Comment:
It looks like this is the only thing in this file that needs to change. Can you revert all of the other changes.